src/config_test.ts

```diff
@@ -265,24 +265,24 @@ describe('KubeConfig', () => {
             const rc = new RequestContext(testServerName1, HttpMethod.GET);
             await kc.applySecurityAuthentication(rc);
             await kc.applytoHTTPSOptions(opts);
-            const expectedCA = Buffer.from('CADATA2', 'utf-8')
+            const expectedCA = Buffer.from('CADATA2', 'utf-8');
             const expectedAgent = new https.Agent({
                 ca: expectedCA,
                 cert: undefined,
                 key: undefined,
                 passphrase: undefined,
                 pfx: undefined,
                 rejectUnauthorized: false,
```

Comment:
this isn't an actual new vulnerability, codeql is triggering on a test that simulates password auth which has `rejectUnauthorized: false`